frontend/.env.dev

```diff
@@ -0,0 +1 @@
+VITE_API_URL=https://reportvision-ocr-api-dev.azurewebsites.net/
```

Comment:
this should work but will probably just need to parametrized the environment tags eventually.

Reply:
something like `https://reportvision-ocr-api-$ENV.azurewebsites.net/

Reply:
yeah seems like the .env files are not working, putting it directly into the build command works